tests/scripts/task_cpp_unittest.sh

```diff
@@ -32,7 +32,7 @@ export OMP_NUM_THREADS=1
 
 # Build cpptest suite
 python3 tests/scripts/task_build.py \
-    --num-executors "${CI_NUM_EXECUTORS}" \
+    --num-executors "${CI_NUM_EXECUTORS:-1}" \
```

Comment:
actually could you only use the default if the `CI` env variable isn’t set? just so we don’t end up triggering this behavior in CI where num executors should always be set. it also might be easier to put the logic in `task_build.py` and remove the `num-executors` flag

Reply:
Sure, see what you think to the latest commit :)